test/unit/middleware/setLocals.js

```diff
@@ -51,23 +38,14 @@ describe('setLocals', () => {
           },
           get: getNull,
         };
-        res = {
-          locals: {
-            backLink: {},
-          },
-        };
+        res = { locals: { backLink: {} } };
```

Comment:
Just a bit of a tidy up, trying to reduce the length of the file as much as anything